Source/WebKit/UIProcess/Extensions/WebExtensionContext.h

```diff
@@ -106,6 +106,15 @@ OBJC_CLASS NSEvent;
 OBJC_CLASS NSMenu;
 #endif
 
+namespace WebKit {
+    class WebExtensionContext;
+}
+
+namespace WTF {
+    template<typename T> struct IsDeprecatedWeakRefSmartPointerException;
+    template<> struct IsDeprecatedWeakRefSmartPointerException<WebKit::WebExtensionContext> : std::true_type { };
+}
+
```

Comment:
@geoffreygaren As an API object this has `ref()` and `deref()` from `API::Object` and `WeakPtr` from `IPC::MessageReceiver`. It seems like that should be fine and not need an exception for this?

This is the same situation for other WebExtension classes in `UIProcess/Extensions`, so I'm not sure why this was the only one needing an exception. And the same case for `WebPageProxy`, etc.

Reply:
I'll investigate. One weakness to my approach is that the compilation failure can be triggered by forgetting an #include. I had like 500 to get through, so I didn't pay close attention. Maybe I can fix these with an #include.

Reply:
I think it was just this one. Fixed.